lib/game.rb

```diff
@@ -0,0 +1,86 @@
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2], # top row
+    [3,4,5], # middle row
+    [6,7,8], # bottom row
+    [0,3,6], # left column
+    [1,4,7], # middle column
+    [2,5,8], # right column
+    [0,4,8], # diagonal from top left
+    [6,4,2]  # diagonal from top right
+  ]
+
+  def initialize(player_1=Human.new("X"), player_2=Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def turn_count
+    @board.count{|space| space == "O" || space == "X"}
```

Comment:
You should be able to use `@board.turn_count` without reimplementing the method.